app/src/main/kotlin/com/saltedge/authenticator/features/main/MainActivity.kt

```diff
@@ -1,7 +1,7 @@
 /*
  * This file is part of the Salt Edge Authenticator distribution
  * (https://github.com/saltedge/sca-authenticator-android).
- * Copyright (c) 2019 Salt Edge Inc.
+ * Copyright (c) 2020 Salt Edge Inc.
```

Comment:
Why do you change the year in old files?

Reply:
and each time in a different way, let's go to one type.

Reply:
I changed because it is the new version of file. It is not a simple update or bug fix.